### PR TITLE
Add support for payload image

### PR DESCRIPTION
**Pull request by woshiluo**
This pull request implements the following features.

- Add support for payload image.
- Always set sstc as enabled for NEMU.
- Add xilinx uarlite driver.

### CODE REVIEW

File: prototyper/Cargo.toml [hunk 1]
```diff
@@ -22,10 +22,14 @@ spin = "0.9.8"
...
 
 
 [[bin]]
 name = "rustsbi-prototyper"
 test = false
 bench = false
 
+[features]
+nemu = []
+fw_payload = []
```
Comment (luojia65) [inline prototyper/Cargo.toml:35]:
```suggestion
payload = []
```

Will this name be better?

File: prototyper/src/board.rs [hunk 2]
```diff
@@ -22,6 +22,7 @@ pub(crate) static mut SBI_IMPL: MaybeUninit<
 #[doc(hidden)]
 pub enum MachineConsole {
     Uart16550(*const Uart16550<u8>),
+    UartAxiLte(MmioUartAxiLite),
```
Comment (luojia65) [inline prototyper/src/board.rs:25]:
```suggestion
    UartAxiLite(MmioUartAxiLite),
```

File: prototyper/src/payload.rs [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+    );
+}
+
+pub fn get_fdt_address() -> usize {
+    let addr: usize;
+    unsafe {
+        asm!("la {}, {fdt}", out(reg) addr, fdt = sym raw_fdt, options(nomem));
+    }
+    addr
+}
```
Comment (luojia65) [inline prototyper/src/payload.rs:27]: Is it possible to convert like `raw_fdt as usize` from `extern { fn raw_fdt(); }` declaration?

**Review by luojia65 — APPROVED** [review]

**Review by guttatus — COMMENTED** [review]
nice work！

**Pull request merged**